app/controllers/rescue_requests_controller.rb

```diff
@@ -74,7 +74,9 @@ def apply_triage_status
   end
 
   def apply_medical_triage_status
+    prev_status = @request.medical_status&.name || "(no status)"
     if @request.update(medical_status_id: params[:status_id])
+      @request.case_notes.create(user_id: current_user.id, medical: true, content: "Changed status from #{prev_status} to #{@request.medical_status.name}")
```

Comment:
If you create this with `user_id: nil` instead, it'll be attributed to System. You can mention the user in the text instead.

Reply:
Done